dotCMS/src/main/java/org/apache/velocity/runtime/VelocimacroFactory.java

```diff
@@ -206,11 +208,14 @@ else if (libfiles instanceof String)
                              twonk.modificationTime = template.getLastModified();
                              libModMap.put(lib, twonk);
                          }
+                         catch(ResourceNotFoundException rnse){
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Either log or rethrow this exception. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1166)
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Move this "catch" on the same line that the previous closing curly brace. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3ARightCurlyBraceSameLineAsNextBlockCheck)